models/dave_robot_models/launch/upload_robot.launch.py

```diff
@@ -128,10 +138,36 @@ def generate_launch_description():
 
     nodes = [tf2_spawner, gz_spawner]
 
+    # Include the Rexrov launch file if the namespace is "rexrov"
+    rexrov_launch = IncludeLaunchDescription(
+        PythonLaunchDescriptionSource(
+            PathJoinSubstitution(
+                [FindPackageShare("dave_robot_models"), "launch", "rexrov.launch.py"]
+            )
+        ),
+        condition=IfCondition(EqualsSubstitution(namespace, "rexrov")),
+    )
+
+    # Include the Glider launch file if the namespace is "glider_slocum"
+    glider_launch = IncludeLaunchDescription(
+        PythonLaunchDescriptionSource(
+            PathJoinSubstitution(
+                [
+                    FindPackageShare("dave_robot_models"),
+                    "launch",
+                    "glider_slocum.launch.py",
+                ]
+            )
+        ),
+        condition=IfCondition(EqualsSubstitution(namespace, "glider_slocum")),
+    )
+
+    include = [rexrov_launch, glider_launch]
+
```

Comment:
@rakeshv24 Could we have those `ros_gz_bridge` launch files in the each robot's description folder along with the `model.sdf`? I think those ros_gz configurations are more like a descriptions for each robot

Example structure:

- dave_robot_models
  - description
    - glider_slocum
       - model.sdf
       - `ros_gz_bridge.py`
    - rexrov
       - model.sdf
       - `ros_gz_bridge.py`

Reply:
This way, we don't need `IfCondition` and keep `upload_robot.launch.py` unchanged when adding new robots

Reply:
Sure that is a good idea. Let me try that out!

Reply:
@woensug-choi but how do we invoke those `ros_gz_bridge.py` files? Don't we still need to use the `IfCondition` for it?

Reply:
@rakeshv24 something like this?
```
    ros_gz_bridge = [IncludeLaunchDescription(
        PythonLaunchDescriptionSource(
            PathJoinSubstitution(
                [
                    FindPackageShare("dave_robot_models"),
                    "launch",
                    "ros_gz_bridge.py",
                ]
            )
        ),
    )]
```

Reply:
Sounds good!